### DISCUSSION TITLE
Measure power

### DISCUSSION THREAD

**Discussion by Jlezana-75** [fhdm-dev/scoppy#115] [General]
First congratulate the creators. I loved the board..

In the future it would be very interesting to create a second measurement section for the APP to measure the power of a signal.

Measure the angle between CH1 and CH2, Multiplications, additions and subtractions and add a real power calculation between the measurement of channel 1 and 2 (V x I x Cos fi= W), where fi is the phase angle.

A Name could be assigned to CH1 and CH2 for this purpose (ch1 = V CH2=I)

It would be really cool to get these calculations, it would help a lot.

Greetings from Chile
Jorge

**Comment by fhdm-dev** [comment]
Hi Jorge,

I've actually been working recently on adding a math channel to the app so hopefully you'll see the addition and multiplication functionality soon.

I'll see what I can do about the real power calculation measurement. That might take a bit longer because the app currently doesn't measure phase angle between waveforms.

**Comment by Jlezana-75** [comment]
What excellent news, we will be attentive to new future modifications.
Congratulations again for this project.

Greetings from Chile.
